trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -882,11 +897,18 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
                              'increasing the bin size.') % my_vbins.size)
 
                     # the virtual bins and their corresponding opacities
-                    my_vbins, vtau = \
-                        tau_profile(
-                            lambda_0, line['f_value'], line['gamma'],
-                            thermb[i], cdens[i],
-                            delta_lambda=dlambda[i], lambda_bins=my_vbins)
+                    if (line['wavelength'].d == 2.1e9): 
```

Comment:
Floating point comparison like this is often unreliable. In addition, I would rather not change the deposition method based solely on the wavelength of the line. I would rather see a new parameter that controls which function is used to do the line deposition, something like `deposition_method` which could default to `voigt`.

Reply:
Added a feature called deposition_method to control line profiles.